README.md

```diff
@@ -114,16 +126,46 @@ make server-init QUIPUCORDS_SERVER_PASSWORD="SuperAdmin1"
 Both of the above commands create a superuser with name `admin` and password of `SuperAdmin1`.
 
 ## Running the Server
+
+### Starting the API Server
+To start the server with API functionality only:
 ```
 make server-static
 make serve
 ```
-To log in to the server, you must connect to http://127.0.0.1:8000 and provide the superuser credentials stated above.
+This will make the API accessible at http://127.0.0.1:8000. 
 
-After logging in, you can change the password and also go to some browsable APIs such as http://127.0.0.1:8000/api/v1/credentials/.
 To use the command line interface, you can configure access to the server by entering `qpc server config`. You can then log in by using `qpc server login`.
 
-If you want to also use the UI, please refer to [quipucords-ui](https://github.com/quipucords/quipucords-ui) for further information.
+### Starting the Celery Worker
+
+Open a separate terminal and run the Celery worker process:
+```
+make celery-worker
+```
+This starts a Celery worker for handling background tasks and asynchronous operations. The Celery worker must remain running alongside the server for proper operation of scheduled tasks and scans.
+
+### Running with UI (Optional)
+If you want to use the UI, you need to install [quipucords-ui](https://github.com/quipucords/quipucords-ui) in a directory at the same level as quipucords. Follow these steps:
+
+1. Clone the **quipucords-ui** repository next to your **quipucords** directory.
```

Comment:
Do the UI and server actually need to have the same parent directory ("same level")? Since we moved all the login page to the UI, and we wiped those assets from the server, I don't _think_ there are any shared assets that would necessitate them living in close proximity on the host filesystem. Or am I forgetting something?

Reply:
good catch. they don't need to be.

Reply:
No, they don't need to be. I will fix.